content/sensu-go/5.18/reference/secrets.md

```diff
@@ -29,9 +29,11 @@ Only Sensu backends have access to request secrets from a [secrets provider][7].
 Sensu backends cache fetched secrets in memory, with no persistence to a Sensu datastore or file on disk.
 Secrets provided via a "lease" with a "lease duration" are deleted from Sensu's in-memory cache after the configured number of seconds, prompting the Sensu backend to request the secret again.
 
-Secrets are only transmitted over a TLS websocket connection.
+Secrets are only transmitted over a transport layer security (TLS) websocket connection.
 Unencrypted connections must not transmit privileged information.
-For agent-side resources, enable TLS/mTLS.
+
+For agent-side resources, you must [enable mutual TLS (mTLS)][13].
```

Comment:
I don't think "agent-side resources" is clear. Which resources are these? I think it means checks, hooks and assets.